### ISSUE TITLE
Command line arguments for setting romfile, bios and the -a parameter seem to guarantee 'Can't initialize Kronos'

### ISSUE THREAD

**Issue by Feldherren** [FCare/Kronos#989]
I've been trying to get command line parameters working (so I can set up a list of games in LaunchBox - important, because Kronos doesn't seem to automatically select appropriate BIOS files from a location), and haven't been having any luck. Via the command line, I've been trying to set BIOS (-b), romfile (-i), and set the emulator to run automatically (-a, I think?). The ROM and BIOS combinations are ones that seem to work fine when specified manually through preferences in Kronos, or when set via command line parameters and NOT using -a.

What's the full name of the game (including region) ?
No particular game - nothing seems to work. I have been testing it with EU Shining Force 3 and fan-translated JP Dragon Force 2, though.

What's your frontend ? (standalone, libretro, ...) ?
Standalone and LaunchBox.

What's your OS ? (windows, linux, OSX, ...) ?
Windows 10

What's your hardware ? (CPU, GPU, ...) ?
Intel i7-6700K, GeForce GTX 980 Ti

What's the storage type of the game (official cd, burned cd or cd image) ?
CD images. Could dig up official physical copies for some of these for testing if required, though.

If applicable, what's the format of your cd image ? (cue/bin, ccd/img, iso, chd, ...)
Cue/bin and CHD.

Are you using any setting different from default ? If so, precise
Not that I can tell, though preferences seems to retain the last manually-set ROM and BIOS files rather than those provided by command line arguments.

Did this game work previously ? Do you remember when ? Is it happening with other games ?
These games function with these ROM/BIOS combinations when not using -a. Just putting -a in seems to cause them to fail immediately with the error 'Can't initialize Kronos'.

If applicable, provide error messages, logs, screenshots, or whatever additional information that could help debugging this

**Comment by EitchKeii** [comment]
I was having the same issue, after playing around with it I found that putting -a _after_ -i "FILE" seems to work. 

Sadly LaunchBox does not allow you to put a parameter after the file path. A temporary workaround is to have LaunchBox open a batch script instead and launch the emulator from the script. If you'd like to do that, it's pretty simple, just make a .bat file with the contents: `kronos.exe -i %1 -a` and put it in the same folder as kronos.exe

Starting from a batch script instead of the actual emulator creates a new problem, you can no longer close the emulator with ESC, so if that's something you would like to use, you could also create an AutoHotKey script that shuts it off for you.
```
$Esc::
{
    if ProcessExist("kronos.exe")
    {
        run, %comspec% /c taskkill /F /IM kronos.exe /T
    }
    else
    {
        Send, {Esc}
    }
}

ProcessExist(Name)
{
    Process,Exist,%Name%
    return ErrorLevel
}
```

If you want to use the AHK script, I'd recommend tossing it in your startup folder, as otherwise, it'd probably be pretty annoying to deal with.

**Comment by Feldherren** [comment]
That would explain the issue.

As an aside, seems like Launchbox already supports a placeholder for ROM files (%romfile%), so we may not need to jump through hoops to put the -a at the end (posted at them about it, got a reply pretty quickly).

**Comment by Feldherren** [comment]
Hm, no. I've tested and I'm still getting issues just running a shortcut with the following:

G:\Emulation\Kronos_2.1.5_64_Bits\kronos.exe -b "D:\Emulation\ROMs\BIOS\Saturn\Sega Saturn BIOS v1.01 (JAP).bin" -i "D:\Emulation\ROMs\Games\Saturn\DRAGON_FORCE_2.chd" -a

Whereas I can confirm that this combination of ROM and BIOS works fine if I select them manually from the emulator.

**Comment by EitchKeii** [comment]
I tried to launch the same game with the same bios using cmd and it worked. 
`kronos.exe -b "bios\Saturn_1.01_J.bin" -i "\\EitchNAS\Roms\Sega\Home Consoles\4. Sega Saturn\JP\Dragon Force II - Kami Sarishi Daichi ni (Rev A).chd" -a`

Not sure what that means, but I don't really have any other ideas.

**Comment by BenjaminSiskoo** [comment]
Hello, thanks for your report. For now, there is a problem with cmd line to launch games. François is busy, he don"t have the time to work on Kronos for now . We are aware about this problem, need to wait :)

**Comment by BenjaminSiskoo** [comment]
Fixed 09.09.2021
